src/components/Button.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as React from 'react';
+import styled from 'styled-components';
+
+type ClickHandler = (el: HTMLElement, ev: MouseEvent) => void;
+type FocusHandler = (el: HTMLElement, ev: MouseEvent) => void;
+type BlurHandler = (el: HTMLElement, ev: MouseEvent) => void;
+type KeyDownHandler = (el: HTMLElement, ev: KeyboardEvent) => void;
+
+type Props = {
+  className: string,
+  ariaLabel: ?string,
+  onClick: ClickHandler,
+  onFocus?: FocusHandler,
+  onBlur?: BlurHandler,
+  onKeyDown?: KeyDownHandler,
+  children?: React$Node,
+};
+
+class Button extends React.Component<Props> {
```

Comment:
Isn't this a common component? Maybe this can be moved into `/components/common`.

Reply:
Just copied CloseLink and put it next to it.

Reply:
The /common pattern does not exist yet, can be a separate change.

Reply:
You're right! Than this should be fine I guess. 👍

Reply:
0 for the component not being part of a common folder.